dev/ci/scripts/ci-coq_lsp.sh

```diff
@@ -9,6 +9,11 @@ git_download coq_lsp
 
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
+if [ -n "${GITLAB_CI}" ]; then
+  export ROCQLIB="$PWD/_install_ci/lib/coq"
+  export ROCQRUNTIMELIB="$PWD/_install_ci/lib/rocq-runtime"
+fi
```

Comment:
we can see in this ci script (and the one for vscoq) that executables not installed in a shared layout with rocq need to be told where to find rocq
this is guarded to GITLAB_CI because ci-common when not GITLAB_CI already exports ROCQLIB/ROCQRUNTIMELIB

Reply:
> IDK if it can be considered a bug

Seems so to me, I'll fix it on my side and update the CI code.